j2k/testData/fileOrElement/methodCallExpression/stringMethods.kt

```diff
@@ -4,10 +4,18 @@ import java.nio.charset.Charset
 import java.util.*
 
 internal class A {
+
+    val FIELD_CONSTANT = "field_constant"
+
     @Throws(Exception::class)
     fun constructors() {
         String()
-        // TODO: new String("original");
+
+        val LOCAL_CONSTANT = "local_constant"
+        val s1 = FIELD_CONSTANT
+        val s2 = LOCAL_CONSTANT
+        val s3 = java.lang.String("original") as String
```

Comment:
The copy constructor is not needed here.

Reply:
Fixed